Makefile.toml

```diff
@@ -29,6 +29,7 @@ env_scripts = [
   set_env RW_TELEMETRY_TYPE "test"
   set_env RW_SECRET_STORE_PRIVATE_KEY_HEX "0123456789abcdef0123456789abcdef"
   set_env RW_TEMP_SECRET_FILE_DIR "${PREFIX_SECRET}"
+  set_env RUST_MIN_STACK 8388608
```

Comment:
@Li0k `risedev` command utilizes the Makefile here. So it has been applied to `risedev test`

Reply:
I suggest not touching this, or at least only set if it's unset.

Reply:
But running `risedev test` locally also fails. I am afraid we cannot accept this.

Reply:
Yes, I am checking the PR you mentioned.